### PR TITLE
Double CRC handling 

### PR DESCRIPTION
**Pull request by githubDante**
The discussion in https://github.com/jmccrohan/pysolarmanv5/issues/62 reveals that some data loggers/inverters apply double CRC over the modbus frame making it unreadable by the standard libraries. Therefore, some bytes must be removed in order the RTU frame to conform to the protocol.

### CODE REVIEW

**Comment by githubDante** [comment]
My tree is messed up too much from what I see. Don't know why all these extra commits are showing up here.

File: pysolarmanv5/pysolarmanv5_async.py [hunk 1]
```diff
@@ -252,6 +254,23 @@ async def _send_receive_modbus_frame(self, mb_request_frame):
         mb_response_frame = self._v5_frame_decoder(v5_response_frame)
         return mb_response_frame
 
+    def _handle_bogus(self, frame: bytes) -> bytes:
```
Comment (jmccrohan) [inline pysolarmanv5/pysolarmanv5_async.py:257]: async already inherits this _handle bogus method from sync version?

Not everyone will be familiar with the discussion in #62, so I wonder should we rename this method to `handle_double_crc_frames()` or similar.

**Comment by davidrapan** [comment]
But this way it is going to call the 'parse_response_adu' twice during every struct.error. Isn't that somehow wasteful?

**Comment by githubDante** [comment]
> But this way it is going to call the 'parse_response_adu' twice during every struct.error. Isn't that somehow wasteful?

The check by length will reduce the calls. Otherwise, yes, it will be called even when the response is a modbus exception.

**Comment by jmccrohan** [comment]
Merged, thanks to both of you for your input on this.

I will tidy up the docs and prepare a new release.

**Pull request merged**